pkgs/applications/science/biology/sambamba/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3, which, dmd, ldc, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sambamba";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "biod";
+    repo = "sambamba";
+    rev = "v${version}";
+    sha256 = "0k5wy06zrbsc40x6answgz7rz2phadyqwlhi9nqxbfqanbg9kq20";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ which python3 dmd ldc ];
+  buildInputs = [ zlib ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp bin/sambamba-${version} $out/bin/sambamba
```

Comment:
In comparison, this project has an `install` target: https://github.com/biod/sambamba/blob/a4b00af81f95956ff8a31b3c60510aaee5d1ed32/Makefile#L114

Reply:
And also a `test` target..

Reply:
The install target is broken because it builds the binary under a different name. I have added a comment reflecting this.